packaging.make

```diff
@@ -1,4 +1,19 @@
 BUILD_CHANNEL?=local
+# note: UNAME_M is overrideable because it is wrong in 32-bit arm container executing natively on 64-bit arm
+UNAME_M ?= $(shell uname -m)
+DPKG_ARCH ?= $(shell dpkg --print-architecture)
+APPIMAGE_ARCH ?= $(shell dpkg --print-architecture)
```

Comment:
As noted elsewhere, dpkg isn't always available, so this will throw errors. That said, as it's ONLY being used for the one new target at the moment (and does NOT throw errors for others), I think it's okay to leave for the moment. I think there'll be a way to clean this up more when it's time to remove the legacy canon/antique1 stuff after this gets some use.

Reply:
made conditional in 9c9aa68